src/libraries/System.Net.Http/src/System/Net/Http/Headers/WarningHeaderValue.cs

```diff
@@ -12,8 +13,8 @@ namespace System.Net.Http.Headers
     public class WarningHeaderValue : ICloneable
     {
         private int _code;
-        private string _agent;
-        private string _text;
+        private string _agent = null!;
```

Comment:
Same here, looks like a private default ctor is forcing this. Can it be removed?

Reply:
Same here, used once and fields set after that